config/base/rbac/role.yaml

```diff
@@ -281,6 +274,13 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - networking.istio.io
+  - security.istio.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
```

Comment:
Is this intentional? :)

Reply:
Yes and no. This is the result of running `make`